packages/theme-patternfly-org/components/sideNav/sideNav.js

```diff
@@ -66,7 +66,7 @@ export const SideNav = ({ groupedRoutes = {}, navItems = [] }) => {
                     className="ws-side-nav-group"
                     onClick={(event) => {
                       // Don't trigger for bubbled events from NavItems
-                      if (!event.target?.href) {
+                      if (!event.target.href) {
```

Comment:
I don't think these are equivalent. I think it should be `if (!event.target || !event.target.href)` for them to be equivalent. The `?` is preventing an exception being thrown if event.target is undefined.

Reply:
@nicolethoen thanks for calling this out - I intentionally changed this, in reviewing I realized the `?.` wasn't really needed, as the `.href` property of `event.target` should always either be defined or return `undefined`